doc/devel/development.md

```diff
@@ -0,0 +1,43 @@
+# Developer guide 
+This document explains how to setup your dev environment. 
+
+## Prerequisite 
+- kubernetes 1.4 
+- go-lang version 1.7
+- docker 
+- glide 
+
+## Setup
+### Step 1
```

Comment:
Most of the setup git stuff is unrelated to etcd operator, and we shouldn't teach people how to use git here. Please remove them and leave only related stuff, e.g. how to glide install, build, etc.